controllers/rating_controller.go

```diff
@@ -88,11 +88,36 @@ func (r *RatingReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		return reconcile.Result{Requeue: true}, err
 	}
 
-	if err := r.CreatePipelineRun(logger, ctx, &instance); err != nil {
-		logger.Error(err, "")
+	instanceDeepCopy := instance.DeepCopy()
+	component := corev1alpha1.Component{}
+	if err := r.Client.Get(ctx, types.NamespacedName{Namespace: instance.Namespace, Name: instance.Spec.ComponentName}, &component); err != nil {
 		return reconcile.Result{}, err
 	}
 
+	repository := corev1alpha1.Repository{}
+	if err := r.Client.Get(ctx, types.NamespacedName{Namespace: instance.Namespace, Name: component.Status.RepositoryRef.Name}, &repository); err != nil {
+		return reconcile.Result{}, err
+	}
+	if !repository.Spec.EnableRating {
```

Comment:
You dont' have to update this rating if its conditioned status has been updated to `ConditionFalse` with message `Rating disabled in component's repository`